server/src/test/java/org/elasticsearch/index/mapper/ParametrizedMapperTests.java

```diff
@@ -296,116 +296,165 @@ private String toStringWithDefaults(ToXContent value) throws IOException {
         assertEquals("mapper [field.sub2] cannot be changed from type [keyword] to [binary]", e.getMessage());
     }
 
     // add copy_to, verify
     public void testCopyTo() {
-        String mapping = "{\"type\":\"test_mapper\",\"variable\":\"foo\",\"required\":\"value\",\"copy_to\":[\"other\"]}";
+        String mapping = """
+            {"type":"test_mapper","variable":"foo","required":"value","copy_to":["other"]}""";
         TestMapper mapper = fromMapping(mapping);
         assertEquals("{\"field\":" + mapping + "}", Strings.toString(mapper));
 
         // On update, copy_to is completely replaced
 
-        TestMapper toMerge = fromMapping(
-            "{\"type\":\"test_mapper\",\"variable\":\"updated\",\"required\":\"value\"," + "\"copy_to\":[\"foo\",\"bar\"]}"
-        );
+        TestMapper toMerge = fromMapping("""
+            {"type":"test_mapper","variable":"updated","required":"value","copy_to":["foo","bar"]}""");
         TestMapper merged = (TestMapper) mapper.merge(toMerge);
-        assertEquals(
-            "{\"field\":{\"type\":\"test_mapper\",\"variable\":\"updated\",\"required\":\"value\"," + "\"copy_to\":[\"foo\",\"bar\"]}}",
-            Strings.toString(merged)
-        );
+        assertEquals("""
+            {"field":{"type":"test_mapper","variable":"updated","required":"value","copy_to":["foo","bar"]}}""", Strings.toString(merged));
 
-        TestMapper removeCopyTo = fromMapping("{\"type\":\"test_mapper\",\"variable\":\"updated\",\"required\":\"value\"}");
+        TestMapper removeCopyTo = fromMapping("""
+            {"type":"test_mapper","variable":"updated","required":"value"}""");
         TestMapper noCopyTo = (TestMapper) merged.merge(removeCopyTo);
-        assertEquals("{\"field\":{\"type\":\"test_mapper\",\"variable\":\"updated\",\"required\":\"value\"}}", Strings.toString(noCopyTo));
+        assertEquals("""
+            {"field":{"type":"test_mapper","variable":"updated","required":"value"}}""", Strings.toString(noCopyTo));
     }
 
     public void testNullables() {
-        String mapping = "{\"type\":\"test_mapper\",\"fixed\":null,\"required\":\"value\"}";
+        String mapping = """
+            {"type":"test_mapper","fixed":null,"required":"value"}""";
         MapperParsingException e = expectThrows(MapperParsingException.class, () -> fromMapping(mapping));
         assertEquals("[fixed] on mapper [field] of type [test_mapper] must not have a [null] value", e.getMessage());
 
-        String fine = "{\"type\":\"test_mapper\",\"variable\":null,\"required\":\"value\"}";
+        String fine = """
+            {"type":"test_mapper","variable":null,"required":"value"}""";
         TestMapper mapper = fromMapping(fine);
-        assertEquals("{\"field\":" + fine + "}", Strings.toString(mapper));
+        assertEquals("""
+            {"field":%s}\
+            """.formatted(fine), Strings.toString(mapper));
```

Comment:
Again, I think it's not worth changing:
```suggestion
assertEquals("{\"field\":" + fine + "}", Strings.toString(mapper));
```